MessengerClone/ChatClient.swift

```diff
@@ -14,7 +14,7 @@ extension ChatClient {
         var components = Components()
         
         components.navigation.channelListRouter = MessengerChatChannelListRouter.self
-        appearance.images.newChat = UIImage(systemName: "square.and.pencil")!
+        appearance.images.pencil = UIImage(systemName: "square.and.pencil")!
```

Comment:
At first, I've changed the image names without realising that they can be replaced and so describing they're use is a better approach. Let me know if it is better here: https://github.com/GetStream/stream-chat-swift/pull/1048/commits/2fd3fc658ed622ec44cc9eb85af85eefb7d361d0

Reply:
I prefer the use description rather than look description. Clients can (and will) customise them a lot